test/apiv2/35-networks.at

```diff
@@ -3,6 +3,26 @@
 # network-related tests
 #
 
-t GET /networks/non-existing-network 404
+t GET networks/non-existing-network 404
+
+if root; then
+  t POST libpod/networks/create?name=network1 '' 200 \
+    .Filename~.*\\.conflist
```

Comment:
Nit (but not worth re-pushing): now that you're creating a named network, you can check for `/network1.conflist`. Same below. Please don't re-push just for this, though.